mevolib/cluster/__init__.py

```diff
@@ -93,4 +94,13 @@ def get_subsets(method: str, seqfile: str, fileformat: str = 'genbank', *args: t
         for key in iter(set_dict):
             for result in output[1:]:
                 set_dict[key].extend(result[key])
-    return set_dict
\ No newline at end of file
+    return set_dict
+
+
+def main():
+    """Default call for Genes module."""     
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input", required = True, help = "No info for you...")
+    parser.add_argument("-n", "--name", required = True, help = "File name needed")
+    args = parser.parse_args()
+    gene_dict = get_subsets('genes', args.input, 'gb', None, None, None, args.name + '.log')
```

Comment:
I would allow users to provide input format in this method:
```suggestion
    parser = argparse.ArgumentParser(
        description="Performs the division of every sequence stored in the input file into subsets by genes"
    )
    parser.add_argument("-i", "--input", required=True, help="Input sequences file")
    parser.add_argument("-f", "--format", default="genbank", help="Input file format")
    parser.add_argument("-n", "--name", required=True, help="Name to give to the log file")
    args = parser.parse_args()
    gene_dict = get_subsets('genes', args.input, args.format, log_file=args.name + '.log')
```
_Note:_ do not forget to submit useful documentation, including `help` strings.